src/c_kzg_4844.c

```diff
@@ -3788,15 +3810,19 @@ C_KZG_RET verify_cell_kzg_proof(
         if (ret != C_KZG_OK) goto out;
     }
 
-    /* Calculate the input value */
+    /* Calculate the value x that identifies the coset
```

Comment:
Seems like you are wrapping the lines a bit too early here? I don't remember if we have precise rules, but we can certainly go up to 80 columns.